website/src/components/GitHubIssues.tsx

```diff
@@ -0,0 +1,185 @@
+  } as React.CSSProperties,
+  lastUpdated: {
+    fontSize: '0.8rem',
+    color: 'var(--ifm-color-emphasis-600)',
+  } as React.CSSProperties,
+  issueItem: {
+    padding: '1rem',
+    borderBottom: '1px solid var(--ifm-color-emphasis-200)',
+    transition: 'background-color 0.2s ease',
+    textAlign: 'left',
+  } as React.CSSProperties,
+  issueItemHover: {
+    background: 'var(--ifm-color-emphasis-100)',
+  } as React.CSSProperties,
+  issueHeader: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'flex-start',
+    marginBottom: '0.5rem',
+    gap: '1rem',
+  } as React.CSSProperties,
+  issueTitle: {
+    fontWeight: 600,
+    color: 'var(--ifm-color-primary)',
+    textDecoration: 'none',
+    flex: 1,
+  } as React.CSSProperties,
+  thumbsUp: {
+    gap: '0.25rem',
+    background: 'var(--ifm-color-emphasis-200)',
+    padding: '0.25rem 0.5rem',
+    borderRadius: '16px',
+    fontSize: '0.85rem',
+    fontWeight: 500,
+    color: 'var(--ifm-color-emphasis-800)',
+  } as React.CSSProperties,
+  issueMeta: {
+    display: 'flex',
+    gap: '1rem',
+    fontSize: '0.85rem',
+    color: 'var(--ifm-color-emphasis-600)',
+  } as React.CSSProperties,
+  issueNumber: {
+    fontWeight: 500,
+    color: 'var(--ifm-color-emphasis-700)',
+  } as React.CSSProperties,
+  authorLink: {
+    color: 'var(--ifm-color-primary)',
+    textDecoration: 'none',
+  } as React.CSSProperties,
+  footer: {
+    padding: '0.75rem 1rem',
+    background: 'var(--ifm-color-emphasis-100)',
+    borderTop: '1px solid var(--ifm-color-emphasis-200)',
+  } as React.CSSProperties,
+  viewAllLink: {
+    color: 'var(--ifm-color-primary)',
+    textDecoration: 'none',
+    fontWeight: 500,
+  } as React.CSSProperties,
+};
+
+function formatDate(dateString: string): string {
+  const date = new Date(dateString);
+  return date.toLocaleDateString('en-US', {
+    year: 'numeric',
+    month: 'short',
+    day: 'numeric'
+  });
+}
+
+export default function GitHubIssues() {
+  const [hoveredIssue, setHoveredIssue] = React.useState<number | null>(null);
+  const pluginData = usePluginData('docusaurus-plugin-github-issues') as GitHubIssuePluginData;
+
+  if (!pluginData || !pluginData.issues || pluginData.issues.length === 0) {
+    return <GitHubIssuesEmpty />;
+  }
+  const issues = pluginData.issues;
+
+  return (
+    <div style={styles.container}>
+      <div style={styles.header}>
+        <div style={styles.headerTitle}>🔥 Popular Feature Requests</div>
+        <span style={styles.lastUpdated}>
+          Last updated: {formatDate(pluginData.lastUpdated)}
+        </span>
+      </div>
+      <div>
+        {issues.map((issue) => (
+          <div
+            key={issue.id}
+            style={{
+              ...styles.issueItem,
+              ...(hoveredIssue === issue.id ? styles.issueItemHover : {})
+            }}
+            onMouseEnter={() => setHoveredIssue(issue.id)}
+            onMouseLeave={() => setHoveredIssue(null)}
```

Comment:
Can we use `XXX:hover` CSS instead to simplify our code?

Reply:
I think we cannot use the pseudo-class in inline style. I will check it!